test_suite/conftest.py

```diff
@@ -21,9 +27,10 @@ def check_markers(host, request):
     # If the host doesn't meet any of the combinations, the test will be skipped.
     run_on_marker = request.node.get_closest_marker('run_on')
     exclude_on_marker = request.node.get_closest_marker('exclude_on')
-    if not run_on_marker and not exclude_on_marker:
+    major_version_marker = request.node.get_closest_marker('major_version')
```

Comment:
I personally don't mind having this new marker for this kind of filtering. However we can only use it for inclusion and not for exclusion. The original idea was to have the capability to use major version filtering in the already existing markers (run_on and exclude_on). So, this would add the freedom of either add a specific minor version of a release, or just filter by major version, or a combination of both for the same marker.

(please note that it's not the same as filtering by "greater than" or "lower than", which should theoretically affect either major versions only or major and minor ones).

Since this is a temp solution until we find a better one e.g. not having to use markers as we discussed last week, I think we are OK for now.

Reply:
Yes you are right, this new marker is useful for inclusion, I thought it would be better to separate it since @jabia99 raised this issue with the major realses that we also might want to have by control when we run tests.
I thought it was better to separate this case for exclution because the regular exclusion is more generic, sometime you'd like to exclude all version beneath a specific one. but with this Major release is more limited